addons/html_editor/static/src/utils/base_container.js

```diff
@@ -19,9 +19,9 @@ export function getBaseContainerSelector(nodeName) {
     return `${nodeName}${suffix}`;
 }
 
-export const baseContainerGlobalSelector = SUPPORTED_BASE_CONTAINER_NAMES.map((name) =>
+export const baseContainerGlobalSelector = `:is(${SUPPORTED_BASE_CONTAINER_NAMES.map((name) =>
     getBaseContainerSelector(name)
-).join(",");
+).join(",")})`;
```

Comment:
Not sure what the gain of this change is, but I'm curious to know.

Reply:
Suppose I want to create a selector like this: `${baseContainerGlobalSelector}:not(.something)`, 
if `baseContainerGlobalSelector` is
```css
p, div.o-paragraph
```

then without is, we would have:
```css
p, div.o-paragraph:not(.something)
```
and with is we would have:
```css
:is(p ,div.o-paragraph):not(.something)
===
p:not(.something), div.o-paragraph:not(.something)
```
which is far more convenient, but if you prefer, I can also use `:is` where I need it and not put it in this exported const

Reply:
No, no, it's fine. I was just curious because it wasn't clear to me the implications that it would have for your needs. Thanks for the explanation!